Automattic/vip-go-mu-plugins
# Add warning if Jetpack couldn't load

## Description
Adds an extra user error if Jetpack was not loaded correctly to make detection and troubleshooting easier.

## Changelog Description

### Plugin Updated: Jetpack loader

Adds an extra user error if Jetpack was not loaded correctly to make detection and troubleshooting easier.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1) Add a return statement at the top of jetpack.php you are loading (similar to as if the version requirements weren't met)
2) Navigate to wp-admin
3) Page still works (although no JP is loaded)
4) Check the logs for the new warning
